docs/reference/manifest/override.md

```diff
@@ -184,3 +185,56 @@ An `<Override>` element expresses a conditional and can be read as an "If ... th
 - [Office versions and requirement sets](../../develop/office-versions-and-requirement-sets.md)
 - [Set the Requirements element in the manifest](../../develop/specify-office-hosts-and-api-requirements.md#set-the-requirements-element-in-the-manifest)
 - [Keyboard shortcuts](../../design/keyboard-shortcuts.md)
+
+## Override element of type RuntimeOverride (preview)
+
+> [!IMPORTANT]
+> This feature is only supported for [preview](../../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) in Outlook on the web and on Windows with a Microsoft 365 subscription. For more details, see [Configure your Outlook add-in for event-based activation](../../outlook/autolaunch.md).
+>
+> Because preview features are subject to change without notice, they shouldn't be used in production add-ins.
+
+An `<Override>` element expresses a conditional and can be read as an "If ... then ..." statement. If the `<Override>` element is of type **RuntimeOverride**, then the `type` attribute is the condition, and the `resid` attribute is the consequent. For example, the following is read "If the type is 'javascript', then the `resid` is 'JSRuntime.Url'." Outlook Desktop requires this element for [LaunchEvent extension point](../../reference/manifest/extensionpoint.md#launchevent-preview) handlers.
```

Comment:
I'm not following what "RuntimeOverride" is. I don't see it in the syntax or example. Is this an element? I notice the other two "types" also have names like this. LocaleTokenOverride for example. If these are real types, we should link to the pages for them. If these are just conceptual names for how the Override element is used, we should just refer to them like "Override for Runtime element" and "Override for locale token" and so on.

Reply:
> I'm not following what "RuntimeOverride" is. I don't see it in the syntax or example. Is this an element? I notice the other two "types" also have names like this. LocaleTokenOverride for example. If these are real types, we should link to the pages for them. If these are just conceptual names for how the Override element is used, we should just refer to them like "Override for Runtime element" and "Override for locale token" and so on.

RuntimeOverride is a type in the official XSD. (go to [MS-OWEMXML](https://docs.microsoft.com/en-us/openspecs/office_file_formats/ms-owemxml/82e93ec5-de22-42a8-86e3-353c8336aa40) and search for "RuntimeOverride". 

The other two are notional "types" that are not formally defined. They are implicitly defined by what their parent element is.  This is inconsistent and confusing. I think we should push back on Zlatko about this design.